internal/routing/handlers/post_handler.go

```diff
@@ -109,7 +114,207 @@ func (handler *PostHandler) CreatePost(w http.ResponseWriter, r *http.Request) {
 			Nickname:          author.Nickname,
 			ProfilePictureURL: "",
 		},
-		Content:   createPostRequest.Content,
-		CreatedAt: createdAt.String(),
+		Content:      createPostRequest.Content,
+		CreationDate: createdAt.String(),
 	}, http.StatusCreated)
 }
+
+func (handler *PostHandler) GetFeed(w http.ResponseWriter, r *http.Request) {
+	// Begin a new transaction
+	tx, transactionCtx, cancel := utils.BeginTransaction(w, r, handler.DatabaseManager.GetPool())
+	if tx == nil || transactionCtx == nil {
+		return
+	}
+	var err error
+	defer utils.RollbackTransaction(w, tx, transactionCtx, cancel, err)
+
+	// Get UserId from JWT token
+	authHeader := r.Header.Get("Authorization")
```

Comment:
Ich glaube hier sollte noch Error-Handling eingebaut werden, wenn der Authorization header nicht entnommen werden konnte. klar kann da "" stehen aber ich glaube es kommt ein error wenn nicht mal irgend ein authorization header gesetzt wurde (i.e. user sendet ein GET ohne jedliche Header)

Reply:
In dem Fall gibt die Methode einen leeren String zurück, den ich einfach entsprechend abhandel